contracts/src/unruggable_memecoin.cairo

```diff
@@ -123,12 +143,65 @@ mod UnruggableMemecoin {
         // ************************************
         // * UnruggableMemecoin functions
         // ************************************
-        fn launch_memecoin(ref self: ContractState) {
-            // Checks: Only the owner can launch the memecoin.
+        // Launches the Memecoin by creating a liquidity pool with the specified counterparty token.
+        // The owner needs to send both MT tokens and the tokens of the chosen counterparty (e.g., USDC).
+        fn launch_memecoin(
```

Comment:
use rust doc style.
/// Description
/// # Arguments
/// - Argument 1: description of argument 1
/// # Returns (if applicable)
/// - ...